testsuite/features/secondary/min_cve_audit.feature

```diff
@@ -89,6 +89,7 @@ Feature: CVE Audit on SLE Salt Minions
     Given I am on the Systems overview page of this "sle_minion"
     When I follow "Software" in the content area
     And I follow "Patches" in the content area
+    And I wait at most 60 seconds until I see "milkyway-dummy-2345" text
```

Comment:
If you tested, LGTM.
If not, consider to included "...refreshing the page" appended.

Reply:
I did not test this.
You mean e.g. that step @srbarrios?

```cucumber
And I wait until I see "milkyway-dummy-2345" text, refreshing the page
```

Reply:
Found it. I will adjust the PR.

Reply:
I cannot find that step.